server/src/service-layer/response-models/UserResponse.ts

```diff
@@ -26,3 +26,7 @@ export interface AllUsersResponse
     CursorPaginatedResponse {
   data?: CombinedUserData[]
 }
+
+export interface BookingIdandUserData extends CombinedUserData {
```

Comment:
nit: can possibly annotate this one (so when we generate code others have documentation) but dont worry about it for now